### ISSUE TITLE
Version 0.26.2 is unrunnable due to esm-only globby

### ISSUE THREAD

**Issue by anomiex** [skovhus/jest-codemods#292]
When I try running `npx jest-codemods`, I wind up with the following error
```
Need to install the following packages:
  jest-codemods
Ok to proceed? (y) 
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated cross-spawn-async@2.2.5: cross-spawn no longer requires a build toolchain, use it instead
/home/anomiex/.npm/_npx/6acdf6878e9bb307/node_modules/jest-codemods/dist/cli/index.js:8
var globby_1 = require("globby");
               ^

Error [ERR_REQUIRE_ESM]: require() of ES Module /home/anomiex/.npm/_npx/6acdf6878e9bb307/node_modules/globby/index.js from /home/anomiex/.npm/_npx/6acdf6878e9bb307/node_modules/jest-codemods/dist/cli/index.js not supported.
Instead change the require of /home/anomiex/.npm/_npx/6acdf6878e9bb307/node_modules/globby/index.js in /home/anomiex/.npm/_npx/6acdf6878e9bb307/node_modules/jest-codemods/dist/cli/index.js to a dynamic import() which is available in all CommonJS modules.
    at Object.<anonymous> (/home/anomiex/.npm/_npx/6acdf6878e9bb307/node_modules/jest-codemods/dist/cli/index.js:8:16)
    at Object.<anonymous> (/home/anomiex/.npm/_npx/6acdf6878e9bb307/node_modules/jest-codemods/bin/jest-codemods.js:2:1) {
  code: 'ERR_REQUIRE_ESM'
}
```
This is due to #267 updating globby from v11 to v13, globby v12 becoming esm-only, and jest-codemods still transpiling to CommonJS for the bin. To make this work, you'll probably have to either downgrade globby or transpile the bin to ESM instead.

**Comment by yogeshgadge** [comment]
This breaks in Angular >=13 which uses globby@12.2.0

**Comment by skovhus** [comment]
Thanks. This should be fixed in 0.26.3
